src/vm/internal/compiler/lex.cc

```diff
@@ -841,10 +846,82 @@ static int skip_to(const char *token, const char *atoken) {
   }
 }
 
+void init_include_path() {
+  svalue_t *ret;
```

Comment:
It has nothing to do with OOP, mostly for better readability.

For this specific snippet, if you move variable declaration close to where it is being used.   I will merge.